core/matching/matchers/json_partial_match.go

```diff
@@ -0,0 +1,132 @@
+package matchers
+
+import (
+	"encoding/json"
+)
+
+var JsonPartial = "jsonpartial"
+
+func JsonPartialMatch(match interface{}, toMatch string) bool {
+	var expected, actual map[string]interface{}
```

Comment:
This may not work if the match or toMatch JSON is an array, and starts with a square bracket. Can you confirm?

Reply:
You right, It doesn't.
I can make this ability, but I'm not sure that described data structure is a valid json.